x-pack/plugins/security_solution/public/management/pages/index.tsx

```diff
@@ -124,9 +126,12 @@ export const ManagementContainer = memo(() => {
         component={EventFilterTelemetry}
         privilege={canReadEventFilters}
       />
-      <Route
+      <PrivilegedRoute
         path={MANAGEMENT_ROUTING_HOST_ISOLATION_EXCEPTIONS_PATH}
         component={HostIsolationExceptionsTelemetry}
+        // remove superuser check in v2 rbac
```

Comment:
[We have a hook](https://github.com/elastic/kibana/blob/27f4f73623961bba7da517fec9557b649aabedf8/x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/hooks.ts#L17) that we should be using everywhere to determine if the user can still access the HIE page. This hook already checks if data exists in cases where the user might have downgraded.

Reply:
95e35d026e1711837421b06b9ccbda8f48e3954f